### PR TITLE
Rename `Inspector` and `Inspector V2` services to match current AWS branding

### PR DESCRIPTION
**Pull request by gdavison**
### Description

Amazon branding for `inspector` and `inspector2` is "Amazon Inspector Classic" and "Amazon Inspector", respectively. Update the wording in documentation and error messages.

### CODE REVIEW

File: GNUmakefile [hunk 1]
```diff
@@ -132,7 +132,6 @@ gen:
 	rm -f internal/sweep/sweep_test.go
 	rm -f names/caps.md
 	rm -f names/*_gen.go
-	rm -f website/allowed-subcategories.txt
```
Comment (ewbankkit) [inline GNUmakefile:135]: ?

Comment (gdavison) [reply GNUmakefile:135]: It's unneeded, since the generator truncates the file

Comment (gdavison) [reply GNUmakefile:135]: Most of these probably are

Comment (ewbankkit) [reply GNUmakefile:135]: 👍

**Review by jar-b — APPROVED** [review]
LGTM 👍

**Pull request merged**